### ISSUE TITLE
[BUG] can't run tracee on gke and do kubernetes

### ISSUE THREAD

**Issue by josedonizetti** [aquasecurity/tracee#2289]
Can't run tracee 0.9.0 on GKE, and DO (only platforms I've tested)

```
containers: failed to mount cgroupv1 controller: failed to mount operation not permitted
libbpf: Error in bpf_create_map_xattr(sys_32_to_64_map):Operation not permitted(-1). Retrying without BTF.
libbpf: map 'sys_32_to_64_map': failed to create: Operation not permitted(-1)
libbpf: permission error while running as root; try raising 'ulimit -l'? current value: 64.0 KiB
libbpf: failed to load object 'embedded-core'
{"level":"fatal","ts":1666558349.710507,"msg":"app","error":"error initializing Tracee: failed to load BPF object: operation not permitted"}

```

**Comment by josedonizetti** [comment]
@rafaeldtinoco this is a major one to be fixed tmr, we really need to be able to run tracee on kubernetes.

**Comment by josedonizetti** [comment]
I enabled the capabilities by pass, and got a different error, we should also look into it:

```
libbpf: Error in bpf_create_map_xattr(sys_32_to_64_map):Operation not permitted(-1). Retrying without BTF.
libbpf: map 'sys_32_to_64_map': failed to create: Operation not permitted(-1)
libbpf: permission error while running as root; try raising 'ulimit -l'? current value: 64.0 KiB
libbpf: failed to load object 'embedded-core'
{"level":"fatal","ts":1666559150.323391,"msg":"app","error":"error initializing Tracee: failed to load BPF object: operation not permitted"}

```

**Comment by rafaeldtinoco** [comment]
Alright, I'm on it now...

**Comment by yanivagman** [comment]
Was it fixed? Still happens with master on GKE

**Comment by josedonizetti** [comment]
Let me check again. After the PR https://github.com/aquasecurity/tracee/pull/2290 I create a custom image, tested it and seemed to work. But now that we have a release it better because I don't have to generate a new image, maybe  I did something wrong there.

**Comment by rafaeldtinoco** [comment]
Please check with: https://github.com/aquasecurity/tracee/pull/2298

**Comment by yanivagman** [comment]
This was fixed by #2298
